docs/build/api/overview.md

```diff
@@ -0,0 +1,77 @@
+---
+sidebar_position: 1
+---
+
+# Introduction to the Fluence Console API
+
+The Fluence Console API provides programmatic access to the decentralized Fluence compute marketplace, allowing developers to find, rent, and manage compute resources without using the web interface. With this API, you can integrate Fluence's decentralized compute capabilities directly into your applications and workflows.
+
+:::info
+All API endpoints are available at:
+
+```bash
+https://vodopad.mainnet.fluence.dev // TODO: switch to api.fluence.dev once it's ready
+```
+
+:::
+
+## What You Can Do with the Fluence Console API
+
+The API enables you to:
+
+1. **Search the Marketplace** - Find compute resources that match your specific requirements including CPU, memory, storage, geographic location, and budget constraints
+2. **Deploy Virtual Machines** - Create and configure VMs with your choice of operating system and network settings
+3. **Manage Resources** - View your active deployments and monitor their status
+
+## Authentication
+
+The Fluence Console API uses API keys for authentication. All requests must include your API key in the Authorization header. You can read more about API keys creation and management in the [Fluence Console Settings guide](../settings/settings.md).
+
+### Using Your API Key
+
+Include your API key in the `Authorization` header with all API requests:
+
+```bash
+curl -X POST https://api.fluence.dev/vms/v3 \
+  -H "Authorization: Bearer <YOUR_API_KEY>"
+```
+
+## Request and Response Format
+
+The Fluence Console API accepts and returns data in JSON format. When making POST requests, set the `Content-Type` header to `application/json`:
+
+```bash
+curl -X POST https://api.fluence.dev/marketplace/offers \
+  -H "Content-Type: application/json" \
+  -H "Authorization: Bearer <YOUR_API_KEY>" \
+  -d '{
+    "basicConfiguration": "cpu-4-ram-8gb-storage-25gb"
+  }'
+```
+
+## Common API Endpoints
+
+The API is organized around these main resource areas:
+
+| Endpoint         | Description                                                         |
+| ---------------- | ------------------------------------------------------------------- |
+| `/marketplace/*` | Endpoints for searching and discovering available compute offerings |
+| `/vms/*`         | Endpoints for deploying and managing virtual machines               |
+
+:::info
+
+You can find detailed information about all API endpoints in the [Fluence Console API Reference](https://vodopad.mainnet.fluence.dev/docs). // TODO: switch to api.fluence.dev once it's ready
+
+:::
+
+## Rate Limits
+
+Currently, the API has no rate limits. // TODO: add rate limits once we have them
```

Comment:
Currently, the API has no rate limits .. does that strike you as a good idea? i'd not advertise the fact, regardless.